components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -152,7 +152,7 @@ export default function OrganizationSelector() {
     return (
         <ContextMenu customClasses="w-64 left-0" menuEntries={entries}>
             <div className={`${classes} rounded-2xl pl-1`}>
-                <div className="py-1 pr-1 flex font-medium whitespace-nowrap max-w-xs overflow-hidden">
```

Comment:
we would usually use [overflow-ellipsis](https://v2.tailwindcss.com/docs/text-overflow#overflow-ellipsis) as well. cc @gtsiolis

Reply:
Yeah, it would be nice if we could use the truncation here.

Reply:
any particular reasons to use `overflow-ellipsis`?

I had this understanding (from [docs](https://tailwindcss.com/docs/text-overflow#truncate))

![image](https://github.com/gitpod-io/gitpod/assets/55068936/1ed5061c-9791-4bdd-8371-da2cd53c1667)

Reply:
we are still on [tailwind 2.x](https://v2.tailwindcss.com/docs/text-overflow#overflow-ellipsis)

Reply:
Whoa, I also didn't notice we also upgrade to 3.* in https://github.com/gitpod-io/gitpod/pull/18525—thanks @AlexTugarev!. I'll close https://github.com/gitpod-io/gitpod/issues/11846, this would unlock a few issues like https://github.com/gitpod-io/gitpod/issues/10004. Thanks, everyone!